src/utils/config.ts

```diff
@@ -117,7 +122,7 @@ export async function findTargetInConfiguration(
     : undefined
 
   if (fallBackTargetJson) {
-    process.logger?.warn(
+    process.logger?.info(
```

Comment:
`targetName` will always be an empty string here

Reply:
got it, fixed